docs/source/tools/ledger-api-test-tool/index.rst

```diff
@@ -0,0 +1,127 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _ledger-api-test-tool:
+
+Ledger API Test Tool
+####################
+
+The Ledger API Test Tool is a command line tool for testing the correctness of
+ledger implementations based on DAML and the :doc:`Ledger API
+</app-dev/ledger-api-introduction/index>`. It’s useful for ledger implementation
+developers, who are using the DAML Ledger Implementation Kit to develop a DAML
+Ledger on top of their distributed-ledger or database of choice.
+
+Use this tool to verify if the Ledger API endpoint conforms to the :doc:`DA
+Ledger Model </concepts/ledger-model/index>`.
+
```

Comment:
Add a paragraph clarifying the status of the tool, which is ALPHA.
- ALPHA: contains a subset of the tests we use internally to test our ledger implementations
- BETA: will be reached once we extended the coverage to include the full set of our test (ADD LINK TO ISSUE)

Tag issue with DLIK label and add to backlog for DLIK: https://github.com/digital-asset/daml/milestone/5

Reply:
+1